docs/src/pages/[platform]/connected-components/liveness/faqs.mdx

```diff
@@ -0,0 +1,25 @@
+import { Fragment } from '@/components/Fragment';
+
+## Frequently Asked Questions
+
+The following issues are presented with corresponding troubleshooting steps. Please check them out in case you run into issues like these -- if the remedies don't work, please file an issue in our GitHub repositories.
+
+### Error: Cognito user pool does not allow access to unauthenticated identities
+
+First, make sure you have followed the [prerequisites for setting up Amplify resources](#step-1-configure-auth). Second, you can also go into the Cognito identity pool configured in your Amplify app, edit it and select the checkbox labeled "Enable access to unauthenticated identities" -- then save the changes. Finally, if using React, ensure that your `aws-exports.js` file exists and is being imported correctly into your app.
```

Comment:
@thaddmt Would this error really show if they didn't have `aws-exports`? Seems like you would get the "Auth not configured" error instead?

Reply:
Yeah that's correct. This should probably more be about ensuring that your aws-exports.js or amplify configuration has references to an identtity pool